feature_apps/src/main/java/cm/aptoide/pt/feature_apps/presentation/ViewModelProvider.kt

```diff
@@ -182,3 +174,6 @@ fun categoryApps(
 fun String.toPackageNameParam() = "package_name=$this"
 
 fun Long.toAppIdParam() = "app_id=$this"
+
+fun String.appendStoreName(storeName: String) =
```

Comment:
Lets have extensions `asSource` for corresponding classes in addition to these on primitives

Reply:
Still can we encapsulate this?